### PR TITLE
Basic structure of mobile knowledge base

### PR DESCRIPTION
**Pull request by ashfurrow**
This PR adds a basic knowledge base for developing iOS software at Artsy, which came out of our Front-End iOS Practice meeting this morning ([notes are here 🔐 ](https://www.notion.so/artsy/July-19-2019-Meeting-Notes-12000fc88fa1480997fb3ac0e7375f04)). The goal is to provide a place where we can accumulate knowledge of how to build iOS software, specifically iOS software at Artsy.

It is a work-in-progress; I've purposefully left some TODO comments to give others a jumping-off point to add their own resources. I couldn't help adding links to docs and blog posts that I think are important, though 😄 The most nebulous section is, and will be, how+why to use Objective-C as a React Native developer. Really exciting times!

### CODE REVIEW

File: resources/mobile/xcode.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Xcode is Apple's IDE for developing, among many other things, iOS applications. You can install Xcode in one of two
+ways:
+
+- [From the App Store](https://apps.apple.com/us/app/xcode/id497799835?mt=12)
+- [From Apple's Developer portal 🔐](https://developer.apple.com/download/more/)
+
+Either works, but beware! If you have App Store automatic updates enabled, you may come in to work one day and find
+that your IDE has changed (which has implications for Swift language and SDK versions). If you ever need a specific
+version of Xcode, check out Apple's developer portal.
```
Comment (sweir27) [inline resources/mobile/xcode.md]: Can you add a link to `Apple's developer portal`?

File: resources/mobile/xcode.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Xcode is really different from VSCode. It's Apple's opinion on what a good IDE should be, and sometimes it's
+uncomfortable at first. Some quick differences:
+
+- Xcode doesn't encourage you to have multiple files open at a time, in tabs.
```
Comment (sweir27) [inline resources/mobile/xcode.md]: These three points are very helpful 😅

File: resources/mobile/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Don't forget to check out the [Front-End iOS Practice](../practices/front-end-ios.md) for less technical, more
+organizational resources.
+
+## Getting Started
+
+There are three distinct ways of developing iOS software at Artsy:
+
+- Writing React Native components in [Emission](https://github.com/artsy/emission) (setup instructions in Readme)
+- Writing Objective-C and Swift, sometimes using on React Native componts _from_ Emission, in
```
[the remark this answers is not in the document]
Comment (ds300) [reply resources/mobile/README.md]: The phrasing of this bullet is kinda hard for me to parse. Does it mean "Writing Objective-C and Swift. Sometimes using them in eigen to work with React Native components imported from emission"?

[the remark this answers is not in the document]
Comment (ashfurrow) [reply resources/mobile/README.md]: You're right – the sentence is unclear. I meant that some React Native components, like `OpaqueImageView` for example, have associated Objective-C code. I almost think of them as "backing" the React components. I'll reword 👍

**Pull request merged**